core-api/src/main/java/com/optimizely/ab/notification/NotificationCenter.java

```diff
@@ -16,9 +16,13 @@
  */
 package com.optimizely.ab.notification;
 
+import com.optimizely.ab.config.Experiment;
```

Comment:
Can you change the license header to say 2017-2018. Let's make sure to do this for the files we touch in the PR